corehq/apps/data_analytics/malt_generator.py

```diff
@@ -47,6 +47,7 @@ def _get_malt_row_dicts(self, user, domain_name, monthspan):
             except Exception as ex:
                 logger.info("Failed to get rows for user {id}, app {app_id}. Exception is {ex}".format
                             (id=user._id, app_id=app_id, ex=str(ex)))
+                continue
```

Comment:
Missed adding `continue` here. Nothing bad happens without this but Reference Exception is raised, and it will be caught and printed in the log